actors/builtin/miner/miner_test.go

```diff
@@ -1307,7 +1308,7 @@ func TestCCUpgrade(t *testing.T) {
 		oldPower := miner.QAPowerForSector(actor.sectorSize, oldSector)
 		expectedFee := miner.PledgePenaltyForContinuedFault(actor.epochRewardSmooth, actor.epochQAPowerSmooth, oldPower)
 		expectedPowerDelta := miner.NewPowerPairZero()
-		actor.applyRewards(rt, expectedFee, big.Zero())
+		actor.applyRewards(rt, bigRewards, big.Zero())
```

Comment:
Non-blocking -- I think the use of applyRewards to set up test state for penalization is wrong.  This PR solves the problem at hand but the brittleness from relying on the production reward function won't go away.